ansible/roles/eessi/tasks/main.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Install CVMFS repo
+  yum:
+    name: https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
+    # NOTE: Can't find any docs on obtaining gpg key
+    disable_gpg_check: true
+- name: Install CVMFS
+  yum:
+    name: cvmfs
+- name: Install EESSI CVMFS config
+  yum:
+    name: https://github.com/EESSI/filesystem-layer/releases/download/latest/cvmfs-config-eessi-latest.noarch.rpm
+    # NOTE: Can't find any docs on obtaining gpg key
```

Comment:
This link does give us a GPG key for the CVMFS repo which helps, but still can't find one for the actual EESSI package. Maybe it's not so bad here since it's coming directly from GitHub?